packages/kbn-plugin-generator/sao_template/template/public/app.js

```diff
@@ -6,7 +6,7 @@ import { render, unmountComponentAtNode } from 'react-dom';
 import { I18nProvider } from '@kbn/i18n/react';
 <%_ } _%>
 
-import 'ui/autoload/styles';
+import 'ui/autoload/all';
```

Comment:
I'm not surprised that `ui/autoload/all` solves this, but I think it's too big of a hammer for what's actually desired here. @elastic/design can we make sure that all of the necessary styles to render the chrome are imported by the `ui/chrome` module itself and don't require plugins to use any of the `ui/autoload/*` modules?